cli/src/program.rs

```diff
@@ -2209,11 +2261,21 @@ fn do_process_program_write_and_deploy(
     buffer_authority_signer: &dyn Signer,
     allow_excessive_balance: bool,
     skip_fee_check: bool,
+    max_retries: Option<usize>,
+    compute_unit_price: Option<u64>,
 ) -> ProcessResult {
     let blockhash = rpc_client.get_latest_blockhash()?;
+    let mut initial_instructions: Vec<Instruction> = Vec::new();
+
+    // Set compute unit price
+    if let Some(compute_unit_price) = compute_unit_price {
+        initial_instructions.push(ComputeBudgetInstruction::set_compute_unit_price(
```

Comment:
it is highly recommended to set `compute-unit-price` and `compute-unit-limit` together. Using default cu-limit may ended up to be overpaying

Reply:
this is resolved , i calculate compute_units by multiplying ixs.len() + 2 with 200k 
then set compute_limit